src/components/snapshots/triggerTables.ts

```diff
@@ -31,11 +34,3 @@ export const triggerTables = [
   'permission_join',
   'file',
 ]
-
-/*
-The following tables have triggers, but *shouldn't* be disabled, as
-they're required for the snapshot import process:
-
-- template
-
-*/
```

Comment:
I remember this being what fixed the problem with importing templates... Wouldn't this cause the issue again?

Reply:
No, because we now only disable the triggers for full re-inits, not template imports -- that's the whole point of this PR! ;)